tools/python_api/src_py/types.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class Type(Enum):
```

Comment:
Is there a way that we don't need to maintain a separate mapping in Python?

Reply:
As discussed in #1189, pybind11 has a memory leakage bug when binding enum class. We can change this `Enum` to binding after the bug is fixed.